libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -109,6 +115,21 @@ const AP_Param::GroupInfo AP_ADSB::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("RF_SELECT",   9, AP_ADSB, out_state.cfg.rfSelect, UAVIONIX_ADSB_OUT_RF_SELECT_RX_ENABLED),
 
+    // @Param: SQUAWK
+    // @DisplayName: Squawk code
+    // @Description: VFR squawk (Mode 3/A) code is a pre-programmed default code when the pilot is flying VFR and not in contact with ATC. In the USA, the VFR squawk code is octal 1200 (hex 0x280, decimal 640) and in most parts of Europe the VFR squawk code is octal 7000. If an invalid octal number is set then it will be reset to 1200.
+    // @Unit: octal
+    // @User: Advanced
+    AP_GROUPINFO("SQUAWK",  10, AP_ADSB, out_state.cfg.squawk_octal_param, ADSB_SQUAWK_OCTAL_DEFAULT),
+
+    // @Param: RF_CAPABLE
+    // @DisplayName: RF capabilities
+    // @Description: Describes your hardware RF In/Out capabilities.
+    // @Values: 0:Unknown,1:Rx UAT only,3:Rx UAT and 1090ES,7:Rx&Tx UAT and 1090ES
+    // @Bitmask: 0:UAT_in,1:1090ES_in,2:UAT_out,3:1090ES_out
+    // @User: Advanced
+    AP_GROUPINFO("RF_CAPABLE",  11, AP_ADSB, out_state.cfg.rf_capable, 0),
```

Comment:
That wasn't my question. Two points:

- code only cares about two bits of this, the ones for *_in*, so I'm not sure what the others are for
- what I asking is if the hardware can only do UAT_in, but the user sets it to UAT_in and 1090ES_in, then will it break hardware? If it doesn't break hardware, then why is this parameter needed? Just set both in code and be done with it.

Reply:
yeah, it is silly. I know. I could remove it. This whole thing is a circus. I put it in just in case. I anticipate them asking about it and ti doesn't hurt to add it now.

Again, the bits set here do not directly correspond to bits set in the encoded packed bytes. They said this works for them..

Your call

Reply:
I've removed the _OUT. will merge after Travis passes